samples/custom_dispatch/cpu/mlp_plugin/CMakeLists.txt

```diff
@@ -37,9 +37,7 @@ set_target_properties(iree_samples_custom_dispatch_cpu_mlp_plugin
 )
 
 add_dependencies(iree-sample-deps
-  iree_samples_custom_dispatch_cpu_mlp_plugin
-  iree_samples_custom_dispatch_cpu_system_plugin)
-
+  iree_samples_custom_dispatch_cpu_mlp_plugin)
```

Comment:
Since we want people to use portable plugins instead of system ones, it'd be good (in the future) to change this all to use portable ones. Then we can also demonstrate multitargeting like we do in the other examples (aarch64 and x86-64, etc). Having the plugin printf isn't really core to the sample and distracts from the happy path we want people to go down for a bulk of plugin uses.